src/Jackalope/ObjectManager.php

```diff
@@ -970,6 +970,18 @@ public function checkout($absPath)
         $this->transport->checkoutItem($absPath); //FIXME: what about pending move operations?
     }
 
+
+    public function addLabel($path, $label, $move)
+    {
+        $this->transport->addLabel($path, $label, $move);
+    }
+
+
+    public function removeLabel($path, $label)
```

Comment:
I've added the methods to the interface and modified the composer.json
But I'm not really sure, if the composer.json is correct. Is it?